doc/Developing/os/Settings.md

```diff
@@ -11,6 +11,28 @@ For example, to set an alternate icon for ios:
 $config['os']['ios']['icon'] = 'fuzzybunny';
 ```
 
+### Ignoring Interfaces
+See also: [Global Ignoring Interfaces Config](../../Support/Configuration.md#interfaces-to-be-ignored)
+
+> These settings are merged with the global settings, so you can only undo global ones with good_if
+
+```yaml
+empty_ifdescr: false # allow empty ifDescr
+bad_if: # ifDescr (substring, case insensitive)
+    - lp0
+bad_if_regexp: # ifDescr (regex, case insensitive)
```

Comment:
Is it not case insensitive depending on the regex? Same for the other two _regexp ones

Reply:
Previous code appended i on the end of the regex, so I left it this way.